clang/test/Driver/x-args.c

```diff
@@ -5,7 +5,3 @@
 // RUN: %clang -fsyntax-only -xc %s -xc++ -fsyntax-only 2>&1 | FileCheck %s
 // RUN: %clang -fsyntax-only %s -xc %s -xc++ -fsyntax-only 2>&1 | FileCheck %s
 // CHECK: '-x c++' after last input file has no effect
-
-// RUN: not %clang_cl /WX /clang:-xc /clang:-E /clang:-dM -- %s 2>&1 | FileCheck --implicit-check-not="error:" -check-prefix=CL %s
```

Comment:
Instead of removing the test, change the `CL` check lines to demonstrate the behavior difference.

Reply:
Added a simple test to demonstrate the availability of `-x` option.